File: visitor/java/Magazine.java
package Visitor;

// UML diyagramındaki Element'ten türer.
// Accept metoduna bulunduğu nesne örneği gönderilir.
// UML diyagramındaki ConcreteElement yapısına denk gelir.
class Magazine extends Book {
    @Override
    public void accept(IBookExportVisitor visitor) {
        visitor.exportVisit(this);
    }
}

File: visitor/python/magazine.py
from book import Book


class Magazine(Book):
    """
    UML diyagramındaki Element'ten türer.
    Accept metoduna bulunduğu nesne örneği gönderilir.
    UML diyagramındaki ConcreteElement yapısına denk gelir.
    """

    def accept(self, visitor):
        visitor.export_visit(self)
